Libplanet/Assets/FungibleAssetValue.cs

```diff
@@ -637,6 +657,10 @@ void ISerializable.GetObjectData(SerializationInfo info, StreamingContext contex
         [Pure]
         public override string ToString() =>
             $"{GetQuantityString()} {Currency.Ticker}";
+
+        public IValue Serialize() => Dictionary.Empty
```

Comment:
If there is no serious backward compatibility concern, I strongly suggest a simple `Bencodex.Types.List` representation, instead of `Bencodex.Types.Dictionary`

Reply:
Does not have a strong opinion about this. I was mocking like other `Serialize()` methods. I'll change it to `Bencodex.Types.List`.